ecs-init/Makefile

```diff
@@ -0,0 +1,38 @@
+.PHONY: generate
```

Comment:
Are you breaking these out into a separate makefile for clarity/readability?  Can you explain why not just expand the top level Makefile?
this is a departure from the ecs-agent: https://github.com/aws/amazon-ecs-agent/tree/master/agent
is the plan to create a similar agent level Makefile?

Reply:
We have our main module in `./ecs-init` (go.mod file) which is now the module path and go command can only be invoked from module path hence the move of these targets to `./ecs-init`

We are doing similar for ecs-agent too - https://github.com/aws/amazon-ecs-agent/commit/bba546eb55a57a4b2c4cbcfdc9897bfc270d3731